ui/common/src/androidMain/kotlin/social/androiddev/common/composables/webview/SignInWebview.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * This file is part of Dodo.
+ *
+ * Dodo is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * Dodo is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Dodo. If not, see <https://www.gnu.org/licenses/>.
+ */
+package social.androiddev.common.composables.webview
+
+import android.annotation.SuppressLint
+import android.graphics.Color
+import android.webkit.WebResourceError
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.viewinterop.AndroidView
+
+@Composable
+actual fun SignInWebView(
+    server: String,
+    onSingedIn: () -> Unit,
+    onFailed: (error: String) -> Unit
+) {
+
+    val signInViewModel = remember(server) { SignInViewModel(server) }
```

Comment:
I left a TODO at the level of the SignInViewModel constructor, because, for the moment, I need to become more familiar with Koin, let alone in a multiplatform project.

If it's something to do right now, I can see what I can do before merging this PR 😬 ?